src/api-to-friendly.ts

```diff
@@ -93,7 +105,7 @@ const apiToFriendlyType = (typeDef: TypeDefinition | TypeDefinition["relations"]
     }
     const relation = relations[0];
     const userSet = (typeDef as any)[relation];
-    apiToFriendlyRelation(relation, userSet, relations, 0, newSyntax);
+    apiToFriendlyRelation(relation, userSet, relations, 0, typeDef?.metadata, newSyntax);
```

Comment:
In this case typeDef looks like:
```
{
  "this": {}
}
```
Your code already handles this, but maybe we can be more explicit and pass in undefined or an empty object here

```suggestion
    apiToFriendlyRelation(relation, userSet, relations, 0, undefined, newSyntax);
```